src/Symfony/Component/Notifier/Bridge/Smsapi/Tests/SmsapiTransportFactoryTest.php

```diff
@@ -26,28 +26,36 @@ public function createProvider(): iterable
         yield [
             'smsapi://host.test?from=testFrom&fast=0',
             'smsapi://token@host.test?from=testFrom',
+            'smsapi://token@host.test?from=testFrom&fast=0&test=0',
         ];
 
         yield [
             'smsapi://host.test?from=testFrom&fast=0',
             'smsapi://token@host.test?from=testFrom&fast=0',
+            'smsapi://token@host.test?from=testFrom&fast=1&test=0',
+            'smsapi://token@host.test?from=testFrom&test=0',
         ];
 
         yield [
             'smsapi://host.test?from=testFrom&fast=1',
             'smsapi://token@host.test?from=testFrom&fast=1',
+            'smsapi://token@host.test?from=testFrom&fast=1&test=1',
+            'smsapi://token@host.test?from=testFrom&test=1',
         ];
 
         yield [
             'smsapi://host.test?from=testFrom&fast=1',
             'smsapi://token@host.test?from=testFrom&fast=true',
+            'smsapi://token@host.test?from=testFrom&fast=true&test=true',
+            'smsapi://token@host.test?from=testFrom&test=true',
```

Comment:
We cannot just extend, we need to add separate yield's

Otherwise they are simply ignored

Reply:
Indeed, I've been to fast on this one. @n3crone can you please send a PR to fix the tests?

Reply:
I will fix them

Reply:
@OskarStark Thank you for help!

Reply:
You're welcome!